common/util/rviz_plugins/autoware_planning_rviz_plugin/README.md

```diff
@@ -0,0 +1,76 @@
+# autoware_planning_rviz_plugin
+
+This package is including jsk code.  
+Note that jsk_overlay_utils.cpp and jsk_overlay_utils.hpp are BSD license.
+
+## Purpose
+
+This plugin displays the path, trajectory, and maximum speed.
+
+## Inputs / Outputs
+
+### Input
+
+| Name                                               | Type                                           | Description                                |
+| -------------------------------------------------- | ---------------------------------------------- | ------------------------------------------ |
+| `/input/path`                                      | `autoware_auto_planning_msgs::msg::Path`       | The topic on which to subscribe path       |
+| `/input/trajectory`                                | `autoware_auto_planning_msgs::msg::Trajectory` | The topic on which to subscribe trajectory |
+| `/planning/scenario_planning/current_max_velocity` | `autoware_planning_msgs/msg/VelocityLimit`     | The topic on which to publish max velocity |
+
+### Output
+
+| Name                                    | Type                            | Description                              |
+| --------------------------------------- | ------------------------------- | ---------------------------------------- |
+| `/planning/mission_planning/checkpoint` | `geometry_msgs/msg/PoseStamped` | The topic on which to publish checkpoint |
+
+## Parameter
+
+### Core Parameters
```

Comment:
Could you separate parameters for each plugin as the following?

MissionCheckPointTool
| Name | Type | Default Value | Description |
....

MaxVelocity
....

Reply:
OK

Reply:
fix https://github.com/tier4/AutowareArchitectureProposal.iv/pull/591/commits/c4e766e86b6696df3d5ac4aa502c752c98e38e97